### PR TITLE
Release 2.1.0: Merge development -> master

### PR DESCRIPTION
**Pull request by davidgrayston**
### Changed
- Button now opens QR in modal
- "Yoti Dashboard" is now referred to as "Yoti Hub"

### Fixed
- LICENSE in root now matches GNU LICENSE in ./yoti/ directory

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,79 +0,0 @@
-FROM php:7.1-apache
```
Comment (emmas-yoti) [inline docker/Dockerfile]: Should there be something in the Release Notes about the rename of the Docker files?

File: yoti/admin/views/yoti/tmpl/default.php [hunk 2]
```diff
@@ -8,14 +8,15 @@
 <div class="row-fluid">
     <div class="span12">
         <form id="yoti-form" method="post" enctype="multipart/form-data" action="<?php echo $_SERVER['REQUEST_URI']; ?>" class="form-horizontal">
+            <?php echo JHTML::_('form.token'); ?>
             <div class="alert">
             <p>
-                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo \Yoti\YotiClient::DASHBOARD_URL; ?>" target="_blank">
-                    <?php echo JText::_('Yoti Dashboard'); ?>
+                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo YotiHelper::YOTI_HUB_URL; ?>" target="_blank">
+                    <?php echo JText::_('Yoti Hub'); ?>
```
Comment (emmas-yoti) [inline yoti/admin/views/yoti/tmpl/default.php]: Dashboard->Hub should probably be in the release notes?

File: LICENSE [hunk 3]
```diff
@@ -1,160 +1,674 @@
...
-
-Audit 

-The Developer acknowledges and agrees that the Provider may, at the reasonable request of the Provider, inspect and have access to such of the Developer’s personnel, facilities and books and records, relevant to the use of the Software, for the purposes of ensuring that the Developer is complying with the terms of these Terms, provided that the Provider provides reasonable advance notice to the Developer of such inspections, which shall take place at reasonable times, unless otherwise requested by any governmental or regulatory body having jurisdiction over the Provider.

-Governing law and jurisdiction
-The Agreement, and any dispute or claim arising out of or in connection with it or its subject matter or formation (including non-contractual disputes or claims) shall be governed by, and construed in accordance with the law of England and Wales. The parties irrevocably agree that the courts of England and Wales shall have exclusive jurisdiction to settle any dispute or claim that arises out of or in connection with these Terms or its subject matter or formation (including non-contractual disputes or claims).
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (emmas-yoti) [inline LICENSE]: This needs mentioning in the changelog

File: docker/Dockerfile (hunk 1 above)
Comment (davidgrayston) [reply docker/Dockerfile]: These are only for local development, CI and demo purposes - @echarrod is it worth mentioning this?

File: LICENSE (hunk 3 above)
Comment (davidgrayston) [reply LICENSE]: The license is already GNU here: https://extensions.joomla.org/extension/yoti/ so will omit there - I'll include in the changelog on GitHub release

Comment (davidgrayston) [reply LICENSE]: Updated PR description

File: yoti/admin/views/yoti/tmpl/default.php (hunk 2 above)
Comment (davidgrayston) [reply yoti/admin/views/yoti/tmpl/default.php]: Updated PR description

File: docker/Dockerfile (hunk 1 above)
Comment (echarrod) [reply docker/Dockerfile]: I think we could add it to our internal dev process wiki, but no worth adding to the README if clients/public don't need to know about them

**Comment by davidgrayston** [comment]
@echarrod I've updated the version for release in 2dee365307b0b8bad16e031f7bf01919d7caa5c9 - there was a duplicate `2.0.0` entry, which I've changed to `2.1.0` https://docs.joomla.org/Deploying_an_Update_Server#Extension

**Comment by davidgrayston** [comment]
@echarrod `./yoti/admin/com_yoti.xml` is copied during installation but shouldn't be included in the packed plugin - fixed in cd8df75

>Note: This has only been an issue since installing from the mounted working copy of the extension.

**Pull request merged**